cli_flags.go

```diff
@@ -39,6 +39,7 @@ const (
 var (
 	noKernelVersionCheckHelp = "Disable checking kernel version for eBPF support. " +
 		"Use at your own risk, to run the agent on older kernels with backported eBPF features."
+	debugBpfHelp       = "Load the BPF program with debugging enabled"
```

Comment:
What is the reasoning to differentiate between `args.verboseMode` and `args.debugBpf`? Using various flags might increase the complexity to be able to debug something. At the moment, `args.verboseMode` does not different between subsystems, e.g. like particular interpreters, why should the eBPF programs be treated differently?

Reply:
Thank you for your feedback, I completely agree with you.